libs/akita-ng-effects/README.md

```diff
@@ -0,0 +1,98 @@
+# Akita Effects
```

Comment:
Add it also to our official docs, please. You can find it in the `docs` folder.

Reply:
It's included now.